pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/actions/ContextMenuAction.kt

```diff
@@ -123,7 +123,7 @@ class ContextMenuAction : RunConfigurationAction() {
             return validFile && validClass
         }
         if (e.place == "ProjectViewPopup") {
-            if (e.getData(CommonDataKeys.PSI_ELEMENT).toString().startsWith("PsiDirectory")) {
+            if (e.getData(CommonDataKeys.PSI_ELEMENT).toString().startsWith("PsiDirectory") && e.getData(CommonDataKeys.PSI_ELEMENT).toString().contains("main")) {
```

Comment:
Won't this include anything that has "main" in it? And what about projects that dont use "src/main/..." but some arbitrary other setup?